### ISSUE TITLE
Selecting Questionnaire plugin for Chr6 displays error

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#5506]
#### Reproduce
- Import the Chr6 EMX questionnaire (will attach later if allowed, else ask me)
- Select the 'Questionnaire' plugin in the 'Plugins' menu

#### Expected
- I see the Chr6 questionnaire and can start data entry

#### Observed
- Freemarker stacktrace:
```
org.springframework.web.util.NestedServletException: Request processing failed; nested exception is org.molgenis.data.validation.MolgenisValidationException: The attribute 'gender' of entity 'chromome6_i_l' can not be null.
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:979)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:858)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
```

**Comment by dennishendriksen** [comment]
We expected this issue to occur in 2.0 since we removed Questionnaire specific hacks from the MOLGENIS data layer.

The proposed solution is along the lines of:
- Introduce AttributeMetadata.nullableExpression
- Allow expressions in EMX attributes sheet --> nillable column
- Update ```nillable``` cells in Chr6 EMX questionnaire with expressions that model the validation behavior described here: https://github.com/molgenis/molgenis/blob/1.21/molgenis-data-validation/src/main/java/org/molgenis/data/validation/RepositoryValidationDecorator.java#L441

**Comment by Mark-de-Haan** [comment]
Fixed by introducing nullable expressions in 5.3.0-SNAPSHOT
